docs/concepts/namespaces.md

```diff
@@ -0,0 +1,35 @@
+---
+id: namespaces
+---
+
+# Namespaces
+
+Namespaces provide users with the capability to publish contracts under their designated namespaces, similar to GitHub's user and organization model.
+
+This feature is currently a work in progress (WIP). To learn more about namespaces, please checkout https://github.com/gnolang/gno/issues/1107.
+
+# Package Path
```

Comment:
This IMO should be changed to import path, but I'll tackle this in another PR where I'll change it globally, both in code and in docs

Reply:
Hello Naming issue #129830911231 again :)